operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -567,9 +567,9 @@ local utils = import 'mixin-utils/utils.libsonnet';
             (
               # We use RSS instead of working set memory because of the ingester's extensive usage of mmap.
               # See: https://github.com/grafana/mimir/issues/2466
-              container_memory_rss{container="ingester"}
+              container_memory_rss{container=~"(ingester|mimir-write|mimir-backend)"}
```

Comment:
Don't hardcode container names, but build the regex from `$._config.container_names` please.

_Same comment applies in other places._